packages/ui/src/utils/UiHelperUtils.ts

```diff
@@ -22,8 +22,12 @@ export const UiHelperUtil = {
     return newUrl.hostname
   },
 
-  getTruncateAddress(address: string, chars: number) {
-    return `${address.substring(0, chars)}...${address.substring(address.length - chars)}`
+  getTruncateString(string: string, chars: number, truncateStart = false) {
```

Comment:
Let's make this function more versatile by changing `truncateStart` from boolean to `location: 'start' | 'middle' | 'end'`. This will also make it more clear when this util is used.